packages/client/hmi-client/src/services/data.ts

```diff
@@ -25,6 +27,8 @@ import {
 import { getFacets as getConceptFacets } from './concept';
 import * as DatasetService from './dataset';
 import { getAllModelDescriptions } from './model';
+// eslint-disable-next-line import/no-cycle
```

Comment:
Should not ignore cyclical imports in my opinion. If it's unavoidable then maybe they should be in the same file.